src/Illuminate/Http/Concerns/InteractsWithInput.php

```diff
@@ -496,17 +496,15 @@ protected function retrieveItem($source, $key, $default)
      */
     public function dd(...$keys)
     {
-        $keys = is_array($keys) ? $keys : func_get_args();
-
-        call_user_func_array([$this, 'dump'], $keys);
+        $this->dump(...$keys);
 
         exit(1);
     }
 
     /**
      * Dump the items.
      *
-     * @param  array  $keys
+     * @param  array|mixed  $keys
      * @return $this
```

Comment:
Mixed can never be used at the same time with another type. Please use mixed here and fix the dd type as well.

Reply:
I have changed it.